runtime/runtime.go

```diff
@@ -159,6 +160,19 @@ func (r *interpreterRuntime) ExecuteScript(
 		}
 	}
 
+	// Ensure entrypoint parameters type is storable
+	if len(epSignature.Parameters) > 0 {
+		for _, param := range epSignature.Parameters {
+			if _, isVoid := param.TypeAnnotation.Type.(*sema.VoidType); !isVoid {
+				if !param.TypeAnnotation.Type.IsStorable(storableResults) {
+					return nil, &ScriptParameterTypeNotStorableError{
```

Comment:
Never realized, but `ExecuteTransaction` is using `newError`, and `ExecuteScript` here is not.
@psiemens do you know why the errors are wrapped in one and not the other?